packages/salesforcedx-sobjects-faux-generator/test/jest/generator/fauxClassGenerator.test.ts

```diff
@@ -1,4 +1,4 @@
-import { TOOLS } from '@salesforce/salesforcedx-utils-vscode/src';
+import { TOOLS } from '@salesforce/salesforcedx-utils-vscode';
```

Comment:
This wasn't your doing or your fault Peter, but `declationGeneratorMocked` is misspelled and should be `declarationGeneratorMocked`.

Since you are making a change to this file, could you replace `declationGeneratorMocked` with `declarationGeneratorMocked`?

Reply:
Will fix